server/game/cards/characters/06/ygritte.js

```diff
@@ -0,0 +1,20 @@
+const DrawCard = require('../../../drawcard.js');
+
+class Ygritte extends DrawCard {
+    setupCardAbilities(ability) {
+        // TODO: Cannot be knelt by card effects.
+        this.persistentEffect({
+            condition: () => this.controlsAnotherWildling(),
+            match: this,
+            effect: ability.effects.addKeyword('stealth')
+        });
+    }
+
+    controlsAnotherWildling() {
+        return this.controller.anyCardsInPlay(card => card !== this && card.getType() === 'character' && card.hasTrait('Wildling'));
```

Comment:
sorry, I was thinking something like findCard() - or should this not scan attachments too?  if so perhaps we need a findCard that skips attachments (kinda like the one youve done)